removing left dividers

tcnichol opened the pull request:
The bug was that if you were a read only user, you saw 2 dividers that were empty on the left.

Now you don't see those 2 dividers, but the 'search' section looks larger than it should.

Not sure how to fix it, so this is draft.
<img width="875" alt="left_panel_issues" src="https://github.com/clowder-framework/clowder2/assets/40038535/6fb7952b-57d5-4907-ae15-7762b5aafd6d">

longshuicy reviewed (changes requested):
Could we just remove the divider altogether? I don't think it will look too bad without it. Thoughts?

tcnichol commented:
@longshuicy i have removed the left dividers and fixed the spacing.

Here is a screenshot of it for logged in (not read only) users
<img width="436" alt="left_tab" src="https://github.com/clowder-framework/clowder2/assets/40038535/eb7c2f89-cbb5-433d-a4bf-882df5fa57ff">

tcnichol commented:
Here is the left drawer for the read only user:
<img width="544" alt="readonly_user_left_tab" src="https://github.com/clowder-framework/clowder2/assets/40038535/c87beb32-4688-4e04-b9a8-a70522b7a4d5">
